js/index.js

```diff
@@ -20,4 +20,41 @@ for (let skill of skillsList){
     let skillsLI = document.createElement('li');
     skillsLI.innerHTML = skill;
     skillsUL.appendChild(skillsLI);
-}
\ No newline at end of file
+}
+
+const messageForm = document.querySelector("[name=leave_message']");
+
+
+messageForm.addEventListener("submit", (event) => {
+    event.preventDefault();
+    let name = event.target.userName.value;
+    let email = event.targer.UsersEmail.value;
+    let message = event.target.usersMessage.value;
+
+    console.log("Name:", name);
+    console.log("Email:", email);
+    console.log("Message:", message);
+    let messageSection = document.getElementbyId("messages");
+    const messageList = messageSection.querySelector("ul");
```

Comment:
You have multiple `ul` elements in your page, JavaScript won't know which is the correct one. I recommend adding an `id` to the `ul` element where you want to insert the results.

Also, the behavior is currently not working as there is no append behavior. You're getting the messages, but never re-insert their content in the ul (`messageList` is unused).